installer.sh

```diff
@@ -29,44 +22,36 @@ source ./.scripts/util.sh
 # For Windows, check that we are running in a MSYS2 UCRT environment
 check_env
 
-PACKAGE_INSTALL_HELP="\
-#  Installs required library packages.
-#  ATM only MSWEBVIEW2 on Windows is fully supported, so we leave this as basic logic.
-#  The installer currently supports CEF 
-#  @todo Expand the logic for multiple OS and embedded browser packages.
+
+SELECT_BROWSER_HELP="\
+#  Selects the browser engine to use
+## Options:
+#      Chromium      Use Chromium Embedded Framework (CEF)
+#      MSWebView2    Use Microsoft WebView2 (Windows only)
 ## Usage:
-#   packages_install\
+#      select_browser <chromium|mswebview2>                               (required)
```

Comment:
NB! Stay consistent with option names and casing:
- chromium
- mswebview2

All lower case.